docs/source/FPGA-to-bitstream/Nextpnr compilation.rst

```diff
@@ -1,7 +1,7 @@
 Nextpnr compilation
 ===================
 
-Compile JSON to FASM by nextpnr <-- bels.txt + pips.txt
+Compile JSON to FASM by nextpnr ← bels.txt + pips.txt
```

Comment:
I'm not sure these lines are super clear anyway. Maybe something more descriptive like

```suggestion
Nextpnr can compile a JSON description of a circuit to FASM (the FPGA Assembly format, describing a concrete list of features on a specific FPGA fabric to be enabled) using the architectural description in bels.txt and pips.txt
```
would be nice. Also don't think it would be entirely unreasonable to drop these lines given it's all explained below - I'd guess they were originally added as placeholders by Jing a few years ago while he fleshed out the docs (and git blame seems to agree)

Reply:
So what do you think, should we still include it or remove it completely? If so, we should either link to FASM or create a footnote since the text in the bracket breaks the reading flow a bit IMHO.

Reply:
There's not really a proper breakdown on the page, so I think it would be reasonable to keep it in this case (I'll comment separately on the other case) - a footnote sounds good!